subsys/bluetooth/mesh/rpl.h

```diff
@@ -0,0 +1,26 @@
+/*  Bluetooth Mesh */
+
+/*
+ * Copyright (c) 2017 Intel Corporation
+ * Copyright (c) 2020 Lingao Meng
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+struct bt_mesh_rpl {
+	uint16_t src;
+	bool  old_iv;
+#if defined(CONFIG_BT_SETTINGS)
+	bool  store;
+#endif
+	uint32_t seq;
+};
+
+extern struct bt_mesh_rpl replay_list[CONFIG_BT_MESH_CRPL];
```

Comment:
`replay_list` should also have the `bt_mesh_` prefix like the functions in this header